local/include/core/ProxyVerifier.h

```diff
@@ -80,6 +82,35 @@ static const std::string YAML_RULE_CONTAINS{"contains"};
 static const std::string YAML_RULE_PREFIX{"prefix"};
 static const std::string YAML_RULE_SUFFIX{"suffix"};
 
+enum class YamlUrlPart { Scheme, Host, Port, Authority, Path, Query, Fragment, Error };
+
+static const swoc::Lexicon<YamlUrlPart> NAME_TO_URL_PART{
+    {{YamlUrlPart::Scheme, {"scheme"}},
+     {YamlUrlPart::Host, {"host"}},
+     {YamlUrlPart::Port, {"port"}},
```

Comment:
It looks like these various string values could be set using `YAML_URL_SCHEME`, `YAML_URL_HOST`, etc.